zp-core/zp-extensions/downloadList.php

```diff
@@ -703,7 +739,7 @@ function printDownloadAlbumZipURL($linktext = NULL, $albumobj = NULL, $fromcache
 			readfile($_zp_downloadfile);
 			exitZP();
 		} else {
-			zp_register_filter('theme_body_open', 'DownloadList::noFile');
+			DownloadList::noFile();
```

Comment:
The new handling of not found files. Now we need to exit ZP, because otherwise a password form will be displayed at this point, no matter what the credential are.